gstools/variogram/variogram.py

```diff
@@ -61,30 +84,35 @@ def estimate_unstructured(field, bin_edges, x, y=None, z=None, sampling_size=Non
 
     return unstructured(field, bin_edges, x, y, z), bin_centres
 
-def estimate_structured(pos, field, direction='x'):
-    """Estimates the variogram of the input data on a regular grid.
+
+def estimate_structured(
+        pos,
+        field,
+        direction='x',
+):
```

Comment:
What's the reasoning behind this style?

Reply:
I think, I had something like this in mind: https://github.com/ambv/black
But in this case it's not necessary to use one line per argument. You're truly right there. Maybe we can use one line for parameters with and one for those without standard value.